changelog.txt

```diff
@@ -1,5 +1,21 @@
 *** Changelog ***
 
+= 1.8.1 - TBD =
+* Fix - Manual orders return an error for guest users when paying with PayPal Card Processing #530
+* Fix - "No PayPal order found in the current WooCommerce session" error for guests on Pay for Order page #605
+* Fix - WooCommerce PayPal Payments & Conditional Discounts By Orion #548
```

Comment:
Could be better to summarize what was actually done instead of a bit confusing titles like this?

e.g. "Fix - Error on order discount by third-party plugins"